docs/02.API-REFERENCE.md

```diff
@@ -10219,19 +10224,24 @@ jerry_value_t
 jerry_exec_snapshot (const uint32_t *snapshot_p,
                      size_t snapshot_size,
                      size_t func_index,
-                     uint32_t exec_snapshot_opts);
+                     uint32_t exec_snapshot_opts,
+                     const jerry_parse_options_t *options_p);
```

Comment:
This seems weird to me. The method "executes" something and we would need to pass "parse" options to it. In addition the comment below mentions "loading options".  So which one is this then? execute/parse/load?

Reply:
You can pass the resource name and user value to this function. These are parts of the `jerry_parse_options_t`, I don't want to introduce another structure for it. Later the resource name could be stored in the snapshot, and only needs to be overwritten if needed. Maybe simple user values can be stored as well.